src/metadata_export.py

```diff
@@ -38,17 +43,13 @@ def metadata_export(sqs_name, job_queue, mongo_dao):
                     export_validator = ExportMetadata(mongo_dao, submission, S3Service())
                     export_validator.export_data_to_file()
                     export_processed += 1
+                    msg.delete()
+                    set_scale_in_protection(False)
```

Comment:
set_scale_in_protection(False). may not be execute if failed to delete a msg. some need swipe the order of these two statement.

Reply:
updated